client/src/components/App.tsx

```diff
@@ -30,10 +30,7 @@ export default function App(props: any) {
   /********************* State Hooks At App Level ******************/
 
   const [dailyData, setDailyData] = useState(async () => undefined);
-  const [activeIndex, setActiveIndex] = useState("daliy");
-  // const [username, userId] = [...props.data];
-  const user_id = props.data.username;
```

Comment:
@nicoleqi7 , you're tests are failing because you've accidentally deleted this line.